src/watts/plugin_openmc.py

```diff
@@ -118,6 +116,20 @@ def __init__(self, model_builder: Callable[[Parameters], None],
         self.show_stdout = show_stdout
         self.show_stderr = show_stderr
 
+    def convert_unit(self, params: Parameters, unit_system, unit_temperature):
```

Comment:
Added this method explicitly to allow the unit test can to check that the unit conversion is correct.